lib/Document/Adapter/Ghostscript.php

```diff
@@ -178,14 +178,14 @@ protected function buildPageCountCommand()
     /**
      * Get the version of the installed Ghostscript CLI.
      *
-     * @return float
+     * @return string
      *
      * @throws \Exception
      */
     protected function getVersion()
     {
         if (is_null($this->version)) {
-            $this->version = (float) Console::exec(self::getGhostscriptCli() . ' --version');
+            $this->version = Console::exec(self::getGhostscriptCli() . ' --version');
```

Comment:
The PhpDoc type should be changed in line 30, too.

Reply:
Fixed